memcached-image/memcached-image.xsl

```diff
@@ -85,31 +85,36 @@
     <source path='obsrepositories:/'/>
   </repository>
 
-  <packages type="image">
-    <package name="ca-certificates"/>
-    <package name="memcached"/>
-    <archive name="memcached-docker-entrypoint.tar"/>
-
-    <!-- extra packages for image -->
-    <xsl:for-each select="param/packages_image/package">
-      <package>
-        <xsl:attribute name="name">
-          <xsl:value-of select="name"/>
-        </xsl:attribute>
-      </package>
-    </xsl:for-each>
-  </packages>
+  <!-- extra packages for image -->
+  <xsl:variable name="packages_image" select="param/packages_image/package" />
+  <xsl:if test="count($packages_image)!=0">
```

Comment:
Doesn't it need to always include memcached, even if the extra packages are empty?